server/src/utility/signup-mail.js

```diff
@@ -6,8 +6,8 @@ const signupMail = async ( firstName, lastName, email) => {
 const transporter = nodemailer.createTransport({
     service: 'hotmail',
     auth: {
-        user: process.env.USER_MAIL,
-        pass: process.env.MAIL_PASSWORD
+        user: process.env.USER_MAIL, //you have to enter your mail-id inside .env file
```

Comment:
Add website link of Hotmail or whatever service u used.

Reply:
okay